pkg/clustermodule/service_test.go

```diff
@@ -81,6 +93,172 @@ func TestService_Create(t *testing.T) {
 			g.Expect(moduleUUID).To(gomega.BeEmpty())
 		})
 	})
+
+	t.Run("creation is executed", func(t *testing.T) {
+		g := gomega.NewWithT(t)
+		_, err := fake_session.MockDefaultSession()
```

Comment:
We'll use it [here](https://github.com/MrDXY/cluster-api-provider-vsphere/blob/pr_unit_test_for_clustermodule/pkg/clustermodule/service_test.go#L146C1-L147) (when you try to create a cluster module)
https://github.com/MrDXY/cluster-api-provider-vsphere/blob/4d2c987b67375a803280b248ef361469d5f870f6/pkg/clustermodule/service_test.go#L146C1-L147

My consideration here is to cache a session and use it for upcoming tests rather than create a session when using it. But it's also an option, only execute `vcsim.NewBuilder()...`, and let the case create a session.